src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +124,57 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         }
         return tagSet;
     }
+
+    /**
+     * Parses a {@code String dateTime} into an {@code LocalDateTime}.
+     *
+     * @throws ParseException if the given {@code dateTime} is invalid.
+     */
+    public static LocalDateTime parseDateTime(String dateTime) throws ParseException {
+        requireNonNull(dateTime);
+        String trimmedDateTime = dateTime.trim();
+
+        DateTimeFormatter format = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
```

Comment:
this one do you think we need `public static final DateTimeFormatter`

Reply:
I think it is not needed for now as this is the only usage of the DateTimeFormatter in the ParserUtil file. Maybe can consider this suggestion during a code refactor?